initfiles/bin/init_thor.in

```diff
@@ -143,8 +147,14 @@ while [[ 1 ]]; do
         fi
     fi
 
-    log "thormaster cmd : $instancedir/thormaster_$THORNAME MASTER=$THORMASTER:$THORMASTERPORT"
-    nohup $instancedir/thormaster_$THORNAME MASTER=$THORMASTER:$THORMASTERPORT 2> /dev/null 1>/dev/null &
+    if [ -z ${valgrindOptions} ]; then
```

Comment:
I generally use quotes around the variable, as in
```
if [ -z "${valgrindOptions}" ]; then
```
But I think it works either way.